src/components/Tabs/components/TabMeasurer/tests/TabMeasurer.test.tsx

```diff
@@ -0,0 +1,87 @@
+import * as React from 'react';
+import {mountWithAppProvider} from 'test-utilities';
+import {noop} from '@shopify/javascript-utilities/other';
+import TabMeasurer from '../TabMeasurer';
+import Tab from '../../Tab';
+import Item from '../../Item';
+
+describe('<TabMeasurer />', () => {
+  const mockProps = {
+    tabToFocus: 0,
+    activator: <Item id="id" focused />,
+    selected: 1,
+    tabs: [
+      {
+        id: 'repeat-customers',
+        content: 'repeat-customers',
+      },
+      {
+        id: 'prospects',
+        content: 'prospects',
+      },
+    ],
+    siblingTabHasFocus: false,
+    handleMeasurement: noop,
+  };
+
+  describe('tabToFocus', () => {
+    it('is used to determine correct focused value of Tab', () => {
+      const tabMeasurer = mountWithAppProvider(<TabMeasurer {...mockProps} />);
+      const tabs = tabMeasurer.find(Tab);
+      expect(tabs.first().prop('focused')).toBe(true);
+      expect(tabs.last().prop('focused')).toBe(false);
+    });
+  });
+
+  describe('siblingTabHasFocus', () => {
+    it('passes siblingTabHasFocus to Tab', () => {
```

Comment:
Nitpick: `gets passed to <Tab />` maybe for clarity and to not repeat `siblingTabHasFocus `? 😄